sdk/test-utils/recorder/test/node/utils.spec.ts

```diff
@@ -150,11 +156,18 @@ describe("NodeJS utils", () => {
         "../../../rush.json": ""
       });
 
+      // If rollup bundle is used to execute the tests - `dist-test/index.node.js`, `recordings` folder is present one level above.
       mockRequire(`../recordings/${filePath}`, {
         // We won't be testing whether MD5 works or not.
         hash: "same old hash"
       });
 
+      // If the dist-esm files are used to execute the tests - `dist-esm/test/node/utils.spec.js`, `recordings` folder is present three levels above.
+      mockRequire(`../../../recordings/${filePath}`, {
+        // We won't be testing whether MD5 works or not.
+        hash: "same old hash"
```

Comment:
Instead of mocking the files at the hard-coded 1 level - 3 levels above, we can pinpoint the recordings folder and mock accordingly.